### PR TITLE
Experimental Kahoot workshop

### PR DESCRIPTION
**Pull request by sampoder**
Hiii! This is an experimental Kahoot workshop.

Demo is at: https://hackclub.zoom.us/rec/share/jZwcyZKGzn-36nVZtYOf2Bnv1ZY-mAJxtlnzBSND1pt06Rfv0BJym2VWeYyx01k2.Gy5DIJsbbdxvIJF5

Passcode: #1i?mf9Q 

In the meantime here's a screenshot:

Screenshot 2020-12-31 at 11 45 10 PM

The slideshow requires the PR: https://github.com/hackclub/workshop-deck-playground/pull/3

### CODE REVIEW

File: workshops/kahoot/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+Run the slideshow through, make sure to give the attendees time to type out their code. 
+
+When you reach the `Let's give it a go!` slide, pass your students the Kahoot code and let them run their programs. 
+Load up Kahoot's reports tab and show the student's scores.
+
+Then comes the challenge section. Begin by doing as the slides instruct. However, if you find the students are struggling load up a [repl.it](https://repl.it) 
+repl with the following code:
+
+```javascript
+const Kahoot = require("kahoot.js-updated");
```
Comment (faisalsayed10) [inline workshops/kahoot/README.md:59]: Can you explain a little about what this code does?

**Review by faisalsayed10 — CHANGES_REQUESTED** [review]
I think it's good! 🚀 
Just one change below and thats it!

File: workshops/kahoot/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+![Game PIN highlighted](https://cloud-lewnc6ao9.vercel.app/0screenshot_2020-12-10_at_10.45.27_pm.png)
+
+Repeat this two more times.
+
+Once you have your game codes you are ready.
+
+## Executing
+
+On the day, load the slideshow from [here](https://workshop-deck-playground-git-add-kahoot-hacking.hackclub.dev/kahoot/0).
```
Comment (faisalsayed10) [inline workshops/kahoot/README.md:48]: The slideshow link in giving me a 404

Comment (sampoder) [reply workshops/kahoot/README.md:48]: try: https://workshop-deck-playground-git-add-kahoot-hacking.hackclub.dev/kahoot

File: workshops/kahoot/README.md (hunk 1 above)
Comment (sampoder) [reply workshops/kahoot/README.md:59]: sure thing

**Review by faisalsayed10 — APPROVED** [review]
Epic!

**Comment by MatthewStanciu** [comment]
oh my god this is so cool

**Pull request merged**